package.json

```diff
@@ -2244,6 +2244,12 @@
                     "type": "boolean",
                     "default": true,
                     "description": "%vscode-docker.config.docker.enableDockerComposeLanguageService%"
+                },
+                "docker.diagnosticLogging": {
```

Comment:
If we're actually intending to merge this, I would rely on VSCode's new log level setting instead, along with the log output channel.

Reply:
Updated to use the LogOutputChannel implementation